azure-pipelines.yml

```diff
@@ -120,3 +120,4 @@ jobs:
       - powershell: eng/validation/update-channel.ps1
           -maestroEndpoint https://maestro-prod.westus2.cloudapp.azure.com
           -barToken $(MaestroAccessToken)
+        displayName: Move build to 'Latest' channel 
```

Comment:
There's already a displayName for this one in LN 108

Reply:
If you expand the "Push Builds to 'Latest' channel" in https://dnceng.visualstudio.com/internal/_build/results?buildId=61056 you'll see this step is shown as "Powershell" only. In this test build you can see it displays "Move build to 'Latest' channel" https://dnceng.visualstudio.com/internal/_build/results?buildId=61149

Reply:
I missed that this is for the step. doh.